contrib/hack/makeKubectl.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# For now pull down a pre-built 'kubectl' image that has the latest
+# build. We need this version in order to pick-up some recent fixes.
+# Once we have an official image we can use then we can delete this section
+docker pull duglin/kubectl
+docker tag duglin/kubectl kubectl
+
+exit 0
+
+# This is the real code we'll keep.
```

Comment:
The previous section can be deleted when the regular (released?) version of kubectl that has the fixes we need is available.  At that time this section will be the real code we'll want to run.